code-challenges.test.js

```diff
@@ -22,12 +22,38 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
-
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("evenlyDivisibleByThreeOrNot", () => {
+
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("logs decides if the number is evenly divisble by three or not", () => {
+  
+      //an expect method, nested within the test block, calling on the evenlyDivisibleByThreeOrNot() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(evenlyDivisibleByThreeOrNot(15)).toEqual("15 is divisible by three")
+      expect(evenlyDivisibleByThreeOrNot(0)).toEqual("0 is divisible by three")
+      expect(evenlyDivisibleByThreeOrNot(-7)).toEqual("-7 is not divisible by three")
+    })
+  })
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes in an number and returns the number wheather is divisible by three or not
+const evenlyDivisibleByThreeOrNot = (number) => {
+  //create a if statement see if the "parameter's modular is not equal 0" true or not
+    if (number % 3 !== 0) {
+      //if it's ture then return the number is not divisible by three
+        return number + ' is not divisible by three'
+      //else then return the number is divisible by three  
+    }else{
+        return number + ' is divisible by three'
+    }
+  }
+//logging the fuction and pass in the num1
+console.log(evenlyDivisibleByThreeOrNot(num1));
+
```

Comment:
You shouldn't have a function invocation here. Your test should call the function.